### ISSUE TITLE
Version constraints

### ISSUE THREAD

**Issue by stevenbrookes** [auxmoney/OpentracingBundle-core#70]
Hi

First off just a thnk you for this great looking set of bundles  - just what I was looking for.

I'm trying to install the following bundles 

auxmoney/opentracing-bundle-jaeger
auxmoney/OpentracingBundle-Monolog
auxmoney/OpentracingBundle-Guzzle

but there are version mismatches between them as reported by composer

 - Root composer.json requires auxmoney/opentracing-bundle-monolog ^0.4.5 -> satisfiable by auxmoney/opentracing-bundle-monolog[v0.4.5].
    - auxmoney/opentracing-bundle-monolog v0.4.5 requires auxmoney/opentracing-bundle-core ^0.7 -> found auxmoney/opentracing-bundle-core[v0.7.0, ..., v0.7.7] but the package is fixed to v0.8.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

Basically it looks like monolog and guzzle plugins need 0.7 and jaeger needs 0.8.  Is there a way to install this, or do the older packages need an update to be compatible?

Thanks

Steve

**Comment by cawolf** [comment]
Hi,

thanks for finding this issue. Yes, all the packages need an update to the latest "core" version. We are currently preparing a 1.0 release, which will mitigate this problem a little in the future.

In the meantime, we will prepare PRs to update all our plugin repositories be up-to-date as soon as we find time. If you want to help, you may of course open PRs in the respective repositories as well. :)

**Comment by stevenbrookes** [comment]
Hi - that's good to hear. The main bundles are working very nicely.  Once I'm up to speed on how it all works I'll see if I can do a PR for the Guzzle package.

**Comment by cawolf** [comment]
For guzzle, we already are in the process of fixing the core dependency in this PR: auxmoney/OpentracingBundle-Guzzle#27

**Comment by cawolf** [comment]
We just released all plugins and the core as version 1. Hopefully, you can now use everything as intended!

**Comment by stevenbrookes** [comment]
Excellent news. Thanks
